### PR TITLE
New features and bugfixes

### PR DESCRIPTION
**Pull request by chlebnymaciej**
- blocking/unblocking bikes
- adding/deleting bikes
- test errors fixed
- databasse migrations errors fixed

### CODE REVIEW

File: tests/test_bikesListViews.py [hunk 1]
```diff
@@ -135,5 +131,4 @@ def test_post_bikes_list_bad_request_status(self, factory, admin):
         request.headers = headers
 
         response = bikes_list(request)
-
-        assert response.status_code == 404
+        assert response.status_code == 400
```
Comment (debskipawel) [inline tests/test_bikesListViews.py:134]: Please stay consistent with status codes and change this one to enum :)

**Review by debskipawel — CHANGES_REQUESTED** [review]
Please make minor changes in tests to maintain consistency

File: tests/test_bikesListViews.py (hunk 1 above)
Comment (chlebnymaciej) [reply tests/test_bikesListViews.py]: ok

**Review by debskipawel — APPROVED** [review]
Looking good now

**Pull request merged**